src/ASM/ASMunstruct.h

```diff
@@ -168,7 +168,9 @@ class ASMunstruct : public ASMbase
   //! \brief Remaps element-wise errors from geometry mesh to refinement mesh.
   //! \param[out] errors The remapped errors
   //! \param[in] orig The element-wise errors on the geometry mesh
-  virtual void remapErrors(RealArray& errors, const RealArray& orig) const = 0;
+  //! \param[in] elemErrors If true, map to elements instead of basis functions
+  virtual void remapErrors(RealArray& errors, const RealArray& orig,
+                           bool elemErrors) const = 0;
```

Comment:
Maybe with default value = false such that the method might be invoked as before without change in behaviour